src/status_im/contexts/chat/messenger/messages/transport/events.cljs

```diff
@@ -350,32 +350,49 @@
            (conj set-hash-fxs
                  #(sanitize-messages-and-process-response % response-js false)))))
 
+(defn- image->rpc [image] (set/rename-keys image {:data-uri :dataUri}))
+
 (defn- link-preview->rpc
   [preview]
   (update preview
           :thumbnail
           (fn [thumbnail]
-            (set/rename-keys thumbnail {:data-uri :dataUri}))))
+            (image->rpc thumbnail))))
+
+(defn- status-link-preview->rpc
```

Comment:
how is it different from ->status-link-previews-rpc ?

Reply:
Sorry forgot and duplicated it, Fixed.